src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -784,6 +782,13 @@ function histogramSelector(publicAPI, model) {
     });
   }
 
+  function createFieldData(fieldName) {
+    return Object.assign(
+      model.fieldData[fieldName] || {},
+      model.provider.getField(fieldName),
+      scoreHelper.defaultFieldData());
```

Comment:
No, they aren't. For future-proofing, when I think about swapping the order, I'm not sure the fieldProvider would have any information that should override the scoreHelper, though. Think it's OK to leave it?